package.json

```diff
@@ -28,7 +28,7 @@
     "paralleljs": "^0.2.1",
     "simd": "^2.0.0",
     "string_format": "^0.0.6",
-    "three": "^0.80.0",
+    "three": "^0.82.0",
```

Comment:
Is the change intentionnal?
I don't mind upgrading three.js, but you should write about this change in the commit message